aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -260,6 +293,31 @@ The `loadBootResource` function can return a URI string to override the loading
 
 The `{TARGET FRAMEWORK}` placeholder is the target framework moniker (for example, `net7.0`). The `{VERSION}` placeholder is the shared framework version (for example, `7.0.0`).
 
+:::moniker range=">= aspnetcore-8.0"
+
+```html
+<script src="{BLAZOR SCRIPT}" autostart="false"></script>
+<script>
+  Blazor.start({
+    webAssembly: {
+      loadBootResource: function (type, name, defaultUri, integrity) {
+        console.log(`Loading: '${type}', '${name}', '${defaultUri}', '${integrity}'`);
+        switch (type) {
+          case 'dotnetjs':
+          case 'dotnetwasm':
+          case 'timezonedata':
+            return `https://cdn.example.com/blazorwebassembly/{VERSION}/${name}`;
+        }
+      }
+    }
+  });
```

Comment:
I think we do at least need to document somehow that when using `blazor.webassembly.js`, the `webAssembly` property here should be omitted, and each of the options should be "top-level". The `webAssembly`, `server`, and `ssr` properties should only be used when using `blazor.web.js`. It's unfortunate that we don't allow both formats when using `blazor.webassembly.js` and `blazor.server.js`, but I'm afraid it's too late to make that change for .NET 8.

Reply:
It's a bit of a tricker call whether we should make a similar comment about `blazor.server.js`, especially if we don't encourage its use for .NET 8 and beyond.

I mentioned an idea about having a table to toggle between the various scenarios in [this comment](https://github.com/dotnet/aspnetcore/issues/51312#issuecomment-1775747693). I definitely understand the hesitation you expressed offline with it being confusing if the columns are named "Blazor Web", "Blazor WebAssembly", and "Blazor Server".

Do you think it would be fine if we have a table where the columns were named `"blazor.web.js"`, `"blazor.webassembly.js"`, and `"blazor.server.js"`? That way it's a bit clearer that it's the `blazor.*.js` file that matters, not the overall hosting model.

cc @danroth27

Reply:
> I think we do at least need to document somehow that when using blazor.webassembly.js, the webAssembly property here should be omitted, and each of the options should be "top-level". The webAssembly, server, and ssr properties should only be used when using blazor.web.js. 

Ok ... I'll work on that on Wednesday morning.

> Do you think it would be fine if we have a table where the columns were named "blazor.web.js", "blazor.webassembly.js", and "blazor.server.js"?

Yes. I need to understand better what you mean by "table" because we do have tables, ***but*** ...

> users can toggle between viewing the "Blazor Web" and "Blazor WebAssembly"/"Blazor Server" versions of each snippet?

... sounds more like we'd ***pivot the whole document*** on the three app types, Blazor Web App, Blazor Server, Blazor WebAssembly. Versioning doesn't play well with pivots because the docs pivot feature absolutely requires content on every pivot for every version or it 💥. If we show all three, each pivot can have different code examples. It's just that pivots are going to force us into showing Blazor Server content 8.0+.

... and you're right to ping @danroth27 because we were trying not to cover Blazor Server explicitly. Here it seems that we'll be required to do so in some capacity, given that the options are set differently for Blazor Server and it's still supported.

Reply:
I think I can fix this up, calling out the standalone Blazor WASM delta and the new BWA approach without pivots.

... and @danroth27 can let me know if I should keep Blazor Server in the mix here for 8.0+. Either way on that, I think I can fix the gremlins :smiling_imp: in this PR! 😆 I'll get back on it first thing Wednesday morning.